src/Samples/VisualBasic/Analyzers/BasicAnalyzers/BasicAnalyzers/tools/uninstall.ps1

```diff
@@ -47,11 +47,18 @@ foreach($analyzersPath in $analyzersPaths)
     $languageAnalyzersPath = join-path $analyzersPath $languageFolder
     if (Test-Path $languageAnalyzersPath)
     {
-        foreach ($analyzerFilePath in Get-ChildItem $languageAnalyzersPath -Filter *.dll)
+        foreach ($analyzerFilePath in Get-ChildItem -Path "$languageAnalyzersPath\*.dll" -Exclude *.resources.dll)
         {
             if($project.Object.AnalyzerReferences)
             {
-                $project.Object.AnalyzerReferences.Remove($analyzerFilePath.FullName)
+                try
+                {
+                    $project.Object.AnalyzerReferences.Remove($analyzerFilePath.FullName)
+                }
+                catch
```

Comment:
I don't know, as I'm just copying this from other scripts in your repo to make them consistent.
But my guess is that failing to uninstall doesn't help the user, and may break a chain of packages being uninstalled. And the user can see the analyzer in SE and remove it him/herself.